release_dashboard/lib/widgets/conductor_status.dart

```diff
@@ -96,45 +96,40 @@ class ConductorStatusState extends State<ConductorStatus> {
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: <Widget>[
-        Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
+        Table(
+          columnWidths: const <int, TableColumnWidth>{
+            0: FixedColumnWidth(200.0),
+          },
+          children: <TableRow>[
+            for (String headerElement in ConductorStatus.headerElements)
+              TableRow(
+                children: <Widget>[
+                  Text('$headerElement:'),
+                  SelectableText((currentStatus[headerElement] == null || currentStatus[headerElement] == '')
```

Comment:
```suggestion
                  SelectableText((currentStatus[headerElement]?.isEmpty ?? true)
```

Reply:
Could you explain this logic to me, please? It worked, but I am a little confused by it.

Reply:
It's functionally the same, but simpler to read. Since you're working with nullable types, you can rely on `?.` to do the null check, and `??` to have a default return if it is null

Reply:
I get an error on `isEmpty`:
```
The getter 'isEmpty' isn't defined for the type 'Object'.
Try importing the library that defines 'isEmpty', correcting the name to the name of an existing getter, or defining a getter or field named 'isEmpty'.dartundefined_getter
```
`currentStatus[headerElement]` is an object type. How should I convert it to a String? 
I tried `(currentStatus[headerElement] as String).isEmpty ?? true` the true becomes unnecessary at that point.

Reply:
That would be complicated to do, since two of the values are arrays that store the cherrypicks.

Reply:
SGTM